tinymce/settings.py

```diff
@@ -11,7 +11,12 @@
 USE_FILEBROWSER = getattr(settings, 'TINYMCE_FILEBROWSER',
         'filebrowser' in settings.INSTALLED_APPS)
 
+
+
 if 'staticfiles' in settings.INSTALLED_APPS or 'django.contrib.staticfiles' in settings.INSTALLED_APPS:
+    if not settings.STATIC_ROOT: #Fix for Django 1.8 which sets STATIC_ROOT as NONE.
+        settings.STATIC_ROOT = ''
```

Comment:
We should probably add a warning or raise an exception here. It is not a good idea to use tiny_mce without settings the ``STATIC_ROOT``.